google-cloud-tools-plugin/src/com/google/gct/idea/debugger/ui/LogoutDebugProcessDetacher.java

```diff
@@ -0,0 +1,28 @@
+package com.google.gct.idea.debugger.ui;
+
+import com.google.gdt.eclipse.login.common.LoginListener;
+
+import com.intellij.execution.process.ProcessHandler;
+
+/**
+ * Detaches a Cloud Debug session represented by an instance of ProcessHandler on a logout event
+ */
+public class LogoutDebugProcessDetacher<H extends ProcessHandler> implements LoginListener {
+
+  private H processHandler;
+
+  public LogoutDebugProcessDetacher(H processHandler) {
+    this.processHandler = processHandler;
+  }
+
+  @Override
+  public void statusChanged(boolean loggedIn) {
+    if (!loggedIn) {
+      detachCloudDebugProcess();
```

Comment:
No it does not. The notification happens through another channel (i.e. the user clicking on the signout button a confirmation message is shown: [GoogleLogin.java#L643](https://github.com/GoogleCloudPlatform/gcloud-intellij/blob/62aa76b773d82fe84addb02e20122b3f3b6690f9/google-account-plugin/src/com/google/gct/login/GoogleLogin.java#L643)  )